CHANGELOG.md

```diff
@@ -21,6 +21,80 @@
 
 ----------------------------------------------
 
+# 12.1.0 Release notes
```

Comment:
Why is this showing up in the diff?

Reply:
I can't do a merge of master as we need to use this branch for release. In order to earn some time to merge it later I thought to checkout just the CHANGELOG to resolve the conflict. But it got really confused.

A 2 dot diff shows the following:
```
$ git diff master..ac/cancel-async-cb-returns-bool CHANGELOG.md
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 6d40b80d2..d98536e1b 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@

 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Changed the signature of `Realm::async_cancel_transaction` to return a boolean indicating whether the removal of the scheduled callback was successful (true) or not (false). Previously, the method returned void. (PR [#5546](https://github.com/realm/realm-core/pull/5546))

 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
```
Which is correct, but github stubbornly  uses the 3 dot diff and all breaks....... So annoying

Reply:
I guess I have to revert it and it'll be fixed later with a merge.

Reply:
You can probably just rebase your branch on top of Core master or merge core master into this branch. We won't be using this branch for the release since there are more fixes that need to land in Core.